src/extensions/unitstable/Units/Units.tsx

```diff
@@ -146,7 +148,7 @@ const Units: React.FunctionComponent<SortingViewProps & OwnProps> = (props) => {
     }
 
     return (
-        <div style={{'width': '100%'}}>
+        <div style={{width: props.width || size.width || 300}}>
```

Comment:
I was having some trouble getting the width to be right. Will need to figure out a cleaner way to do this.